### DISCUSSION TITLE
facebook login/registration

### DISCUSSION THREAD

**Discussion by luivis07** [OrchardCMS/OrchardCore#12714] [Q&A] [answered]
Hi all,
I've enabled Facebook Authentication:
![image](https://user-images.githubusercontent.com/4964745/197910453-a0a849f7-59ae-4e68-9f63-494d5a81ae5a.png)

but the login page only shows my other 2 external auth (git and google):
![image](https://user-images.githubusercontent.com/4964745/197910502-87b23c9f-9925-46b9-8e26-4ef1987ecfb8.png)

In addition, the facebook config page only shows this:
![image](https://user-images.githubusercontent.com/4964745/197910574-84984edb-ba59-41da-9746-89871f40a492.png)

I've also tried configuring Facebook by using the instructions here: https://docs.orchardcore.net/en/latest/docs/reference/modules/Facebook/#facebook-login
and adding my appid and secret to the app.settings file but still doesn't wok. Am I missing something?

**Comment by hishamco** [comment]
@MichaelPetrinolis I think you are the one work on this, right?

**Comment by hishamco** [comment]
I presume enabling the facebook module should make the facebook login button show up in the external AUTH providers

**Reply by luivis07** [reply]
I guess there might be a problem because enabling the facebook module doesn't provide a place to add the "appid" and "secret" like google and github module do.

**Reply by MichaelPetrinolis** [reply]
Hi, I will check it. Probably something broke and the option to set the settings is missing

**Comment by MichaelPetrinolis** [comment, marked as answer]
@luivis07 Facebook Login and widgets depends on Facebook Integration feature. There you configure the Application id and the client secret  as shown in the image below.
![image](https://user-images.githubusercontent.com/9623255/198519225-95306c3e-1510-46d7-9b13-6ecc66d64f37.png)
In login settings you configure the call back url. Maybe we should add a hint that you must configure these settings.
